examples/full/go.mod

```diff
@@ -7,7 +7,7 @@ replace github.com/rebuy-de/rebuy-go-sdk/v8 => ../..
 require (
 	github.com/go-chi/chi/v5 v5.1.0
 	github.com/pkg/errors v0.9.1
-	github.com/rebuy-de/rebuy-go-sdk/v8 v8.0.0
+	github.com/rebuy-de/rebuy-go-sdk/v8 v8.2.1
```

Comment:
The version here doesn't really matter because of the `replace` above, but of course it doesn't hurt either.